cypress/e2e/pages/dashboard.pages.js

```diff
@@ -10,9 +13,98 @@ const viewAllStr = 'View all'
 const transactionBuilderStr = 'Use Transaction Builder'
 const safeAppStr = 'Safe Apps'
 const exploreSafeApps = 'Explore Safe Apps'
+export const copiedAppUrl = 'share/safe-app?appUrl'
 
 const txBuilder = 'a[href*="tx-builder"]'
 const safeSpecificLink = 'a[href*="&appUrl=http"]'
+const copyShareBtn = '[data-testid="copy-btn-icon"]'
+const exploreAppsBtn = '[data-testid="explore-apps-btn"]'
+const viewAllLink = '[data-testid="view-all-link"]'
+const noTxIcon = '[data-testid="no-tx-icon"]'
+const noTxText = '[data-testid="no-tx-text"]'
+const pendingTxWidget = '[data-testid="pending-tx-widget"]'
+const pendingTxItem = '[data-testid="tx-pending-item"]'
+const singleTxDetailsHeader = '[data-testid="tx-details"]'
+
+export function clcikOnTxByIndex(index) {
```

Comment:
misspelling on the method. It should say click

Reply:
Done